docs/community_resources/installing-on-mac-os-and-windows.md

```diff
@@ -71,7 +76,7 @@ The instructions below assumes you are using Apache.
 Download and extract an archive into the location where you want your project root directory to be from [ezplatform.com](https://ezplatform.com/#download-option) (for open-source version) or from the [Support portal](https://support.ez.no/Downloads) (for eZ Enterprise), or clone the [GitHub repository](https://github.com/ezsystems/ezplatform):
 
 ``` bash
-git clone https://github.com/ezsystems/ezplatform .
+git clone https://github.com/ezsystems/ezplatform
```

Comment:
I think the full stop should remain here, to remain consistent with the main install guide.